lib/logstash/filters/cipher.rb

```diff
@@ -157,11 +157,15 @@ def filter(event)
         encryptedKey = Base64.strict_decode64(dataHash["encryptedKey"])
         encryptionContext = eval(dataHash['encryptionContext'])
 
-        decryptedKey = @kms.decrypt(:ciphertext_blob =>  encryptedKey,
-                                   encryption_context: encryptionContext
-                                  )
+        unless @keys_hash.has_key? encryptedKey
+          @logger.warn("YES FETCH")
```

Comment:
Minor improvement: it'd be really nice if this was `@logger.info("Fetching KMS decrypted key for #{encryptedKey}")` in the logs